app.js

```diff
@@ -31,5 +35,6 @@ app.use((err, req, res, next) => {
 
```

Comment:
These can be .send instead of .json. 

The 404 errors should be handled by another error handler for PSQL errors that will be next in the chain.

Reply:
These error handlers still need to be handing to the next error handler rather than in an else chain. eg

```
app.use((err, req, res, next) {
 if (err.status) {
    res.status(err.status).send({ message: err.msg })
    } else { next(err)}
    })
    
app.use((err, req, res, next) {
// handle the next error
})
```